cranelift/codegen/src/isa/pulley_shared/abi.rs

```diff
@@ -4,7 +4,7 @@ use super::{inst::*, PulleyFlags, PulleyTargetKind};
 use crate::isa::pulley_shared::{PointerWidth, PulleyBackend};
 use crate::{
     ir::{self, types::*, MemFlags, Signature},
-    isa::{self, unwind::UnwindInst},
+    isa::{self},
```

Comment:
strange artifact not caught by `rustfmt` maybe? would prefer `isa::self` to `isa::{self}`

Reply:
oh oops I totally didn't notice this at all!